build.gradle

```diff
@@ -1156,6 +1156,8 @@ project('test:system') {
         environment "helmRepository", System.getProperty("helmRepository", "https://charts.pravega.io")
         environment "helmHookImageName", System.getProperty("helmHookImageName", "lachlanevenson/k8s-kubectl")
         environment "tlsEnabled", System.getProperty("tlsEnabled", "false")
+        environment "securityEnabled", System.getProperty("securityEnabled", "false")
+        environment "tlsCertCNName", System.getProperty("tlsCertCNName", "pravega-pravega-controller")
```

Comment:
pravega-pravega-controller can be defaulted to localhost. and System test can pass this parameter "tlsCertCNName" via jenkins.

Reply:
Made default as localhost.